pcsx2/Patch.cpp

```diff
@@ -267,7 +278,15 @@ u32 Patch::LoadPatchesFromString(PatchList* patch_list, const std::string& patch
 
 			if (!current_patch_group.name.empty() || !current_patch_group.patches.empty())
 			{
-				patch_list->push_back(std::move(current_patch_group));
+				// Don't show patches with duplicate names, prefer the first loaded.
+				if (!ContainsPatchName(*patch_list, current_patch_group.name))
+				{
+					patch_list->push_back(std::move(current_patch_group));
+				}
+				else
+				{
+					Console.WriteLn(Color_Gray, fmt::format("Patch: Skipped loading patch '{}' since with duplicate name was already loaded.", current_patch_group.name));
```

Comment:
Accidentally a word?

Reply:
Thanks for catching this, I updated this to be consistent with the other log statement. I had updated one but not the other. 👍
And thanks for the feedback in general.

Reply:
This still sounds strange to me - "since [what?] with duplicate name"?